<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-191</y>
        <width>374</width>
        <height>747</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutScrollArea">
       <item>
        <widget class="QGroupBox" name="groupBoxGeneral">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutGeneral">
          <item row="0" column="0">
           <widget class="QLabel" name="labelStartupPage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Startup Page:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxStartUpPage">
            <item>
             <property name="text">
              <string>Start Page</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Home Page</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelHomePage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Home Page:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEditHomePage"/>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButtonUseCurrent">
            <property name="text">
             <string>Use Current</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxEnableShortcuts">
            <property name="text">
             <string>Enable Keyboard Shortcuts.</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelPrompt">
            <property name="text">
             <string>Prompt Before:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxPromptCloseBrowser">
            <property name="text">
             <string>Closing Browser on Escape</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxPromptCloseTabs">
            <property name="text">
             <string>Closing Tabs on Exit</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxPromptStreamMedia">
            <property name="text">
             <string>Switching to Carrier Network for 
 Streaming Media</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxWebContent">
         <property name="title">
          <string>Web Content</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutWebContent">
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxEnableJS">
            <property name="text">
             <string>Enable JavaScript</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBoxBlockPopups">
            <property name="text">
             <string>Block Popups</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxEnableMedia">
            <property name="text">
             <string>Enable Embedded Media</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="labelFontSize">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Default Font Size:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QComboBox" name="comboBoxFontSize">
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>17</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="labelTextEncoding">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Default Text Encoding:</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QComboBox" name="comboBoxTextEncoding">
            <item>
             <property name="text">
              <string>ISO-8859-1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UTF-8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GB18030</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GB2312</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GBK</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BIG5</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBoxAutoTextEncoding">
            <property name="text">
             <string>Auto-Detect Text Encoding</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxLoadImages">
            <property name="text">
             <string>Load Images</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxPrivacy">
         <property name="title">
          <string>Privacy and Security</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutPrivacy">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxAcceptCookie">
            <property name="text">
             <string>Accept Cookies</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxEnableGeo">
            <property name="text">
             <string>Enable Geolocation</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonGeoPermissions">
            <property name="text">
             <string>Geolocation Permissions</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelClear">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Clear Browsing Data:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBoxClearPasswords">
            <property name="text">
             <string>Passwords</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="checkBoxClearHistory">
            <property name="text">
             <string>History</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="checkBoxClearCookies">
            <property name="text">
             <string>Cookies</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="checkBoxClearCache">
            <property name="text">
             <string>Cache</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QCheckBox" name="checkBoxClearPushContent">
            <property name="text">
             <string>Pushed Content</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QPushButton" name="pushButtonClearNow">
            <property name="text">
             <string>Clear Now</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonResetDefault">
        <property name="text">
         <string>Reset to Default</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>256</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
